dwave/cloud/api/exceptions.py

```diff
@@ -59,3 +59,7 @@ class InternalServerError(RequestError):
 
 class RequestTimeout(RequestError):
     """API request timed out"""
+
+
+class ResourceDeprecationWarning(FutureWarning):
```

Comment:
By subclassing `FutureWarning`, we're making sure deprecation warnings are not silenced by default. Might revisit this later.